Src/Cheatsheet/Cheatsheet.md

```diff
@@ -167,7 +167,7 @@ NOTE: Requires Markdown Extra. See http://michelf.ca/projects/php-markdown/extra
 * Name types using nouns, noun phrases or adjective phrases (AV1708)
 * Don’t repeat the name of a class or enumeration in its members (AV1710)
 * Avoid short names or names that can be mistaken with other names (AV1712)
-* Name methods using verb-object pair (AV1720)
+* Name methods and local functions using verbs or verb-object pairs (AV1720)
```

Comment:
Hmm, when would a local function be named using just a `verb`?

Reply:
I consider LFs similar to methods in that they perform an operation. Examples would then be: Initialize/InitializeFields, Prepare/PrepareDocument, Calculate/CalculateSalesRates, Reassign/ReassignWork etc.